src/Parallel/Phase.cpp

```diff
@@ -19,6 +19,7 @@ std::vector<Phase> known_phases() {
           Phase::BuildMatrix,
           Phase::CheckDomain,
           Phase::Cleanup,
+          Phase::DisableRotationControl,
```

Comment:
What do you think about naming this phase `DisableControlSystems` instead? That way we can use it in the future if any of our systems need to disable any control system

Reply:
So I thought about this a decent amount before submitting the PR and decided to keep it specific because I don't know yet if there will be multiple control systems that need to be disabled simultaneously. If not, then you need a different phase name for them anyway, or some way of tracking state of which one was and should be disabled. My take is that it's less likely to be confusing having the different name. As not-ideal as it is in terms of scaling the code, I think clarity is important since the control system interactions are quite complicated as-is.

Reply:
Ok yes, this makes sense. I think if we do need to add more control systems to be disabled, we can probably do it in such a way that we'd only need one phase, but I haven't thought too much about it. We can cross that bridge when we get to it.

Reply:
Sounds good! Yea, I think it'll be clearer if/when we get there. I don't think for BNS we'll have anything else until we can do a BH+matter in the post-merger. Translation and can just track the global COM instead of the average of the two halves. And for BNS I think we only need Rotation+Translation 🤞